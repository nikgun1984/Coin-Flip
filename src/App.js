import "./App.css";
import FlipCoin from "./FlipCoin";

function App() {
	return (
		<div className="container">
			<div className="row justify-content-center">
				<div className="col-6 text-center mt-5">
					<FlipCoin />
				</div>
			</div>
		</div>
	);
}

export default App;
