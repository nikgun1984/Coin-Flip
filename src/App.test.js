import { render } from "@testing-library/react";
import App from "./App";

// smoke test
it("testing if the app renders without crashing", function () {
	render(<App />);
});

// snapshot test
it("matches snapshot", function () {
	const { asFragment } = render(<App />);
	expect(asFragment()).toMatchSnapshot();
});
