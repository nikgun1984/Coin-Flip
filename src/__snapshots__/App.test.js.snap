// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="row justify-content-center"
    >
      <div
        class="col-6 text-center mt-5"
      >
        <div>
          <h1>
            Let's flip a coin
          </h1>
          <img
            alt="no img found"
            class="Coin-image FlipCoin-hidden"
            data-testid="coin-image"
            src=""
          />
          <div
            class="d-flex justify-content-center"
          >
            <button
              class="btn btn-outline-dark text-bold mt-4"
            >
              Flip MEEE!!!
            </button>
          </div>
          <p
            class="mt-4"
            id="result"
          >
            Out of 0 flips, there have been 0 heads and 0 tails.
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
